Add image size properties to the infoCard component (#1126)
vtex-apps/store-components: docs/InfoCard.md

| `htmlId`                 | `string`            | ID of the container element.                                                                                               | `null`        |
| `imageUrl`               | `string`            | Path to the image used on desktop devices.                                                                                 | `""`          |
| `imageActionUrl`         | `string`            | Redirect URL used when the image component is clicked.                                                                     | `""`          |

> | `imageHeight`            | `number`            | Height of the image that is used to improve CLS while the image is loading.                                                | `null`        |
> | `imageWidth`             | `number`            | Width of the image that is used to improve CLS while the image is loading.                                                 | `null`        |

Reviewer: ```suggestion
| `imageHeight`            | `number`            | Height of the image used to improve [Cumulative Layout Shift (CLS)](https://web.dev/articles/cls) while the image loads.                                                | `null`        |
| `imageWidth`             | `number`            | Width of the image used to improve [Cumulative Layout Shift (CLS)](https://web.dev/articles/cls) while the image loads.                                                 | `null`        |
```
